### DISCUSSION TITLE
Everything works with the new avb subcommand

### DISCUSSION THREAD

**Discussion by emptyteeth** [chenxiaolong/avbroot#155] [Show and tell]
I'm using self compile kernel for my pixel5. Previously, I had to use avbtool.py and Android_boot_image_editor to handle the avb stuff for the custom boot.img and vendor_boot.img, it's a bit messy. 

Now with the new avb subcommand, it's much easier, I just pack them with the stock avb.toml, then patching and sideloading, and everything works at the first try. 

Thank you so so much for the avbroot project!   and BCR,   also RSAF 😆

**Comment by chenxiaolong** [comment]
That's great to hear! Glad everything is working well!
